core/src/test/kotlin/de/fraunhofer/iem/kpiCalculator/core/strategy/AbstractKpiCalculationTest.kt

```diff
@@ -0,0 +1,195 @@
+package de.fraunhofer.iem.kpiCalculator.core.strategy
+
+import de.fraunhofer.iem.kpiCalculator.model.kpi.hierarchy.KpiCalculationResult
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.fail
+
+class AbstractKpiCalculationTest {
+
+    @Test
+    fun calculateKpiEmptyChildren() {
```

Comment:
maybe try out the parametrized tests here to swtitch for strict mode
 https://sam09.github.io/Parameterized-Tests-Kotlin/